src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator.java

```diff
@@ -72,7 +72,7 @@ public int compareTo2(Object aObject) // empty line - ok
      * @param result
      * @return
      */
-    public static <T> Callable<T> callable(Runnable task, T result) // violation
+    public static <T> Callable<T> callable(Runnable task, T result) // violation ''METHOD_DEF' should be separated from previous line.'
```

Comment:
You can remove parameter `Runnable task,` to shorten line.

`public static <T> Callable<T> clb(T r)`